docs/Concepts/Qubernetes/Multicluster.md

```diff
@@ -0,0 +1,360 @@
+      Tm_Url:  2.2.3.4:30482
+      Node_Acct_Addr: 0x9712A5aa48A654e9B117467748eF1F793153E918
+    - Node_UserIdent:  quorum-node2
+      Enode_Url: "enode://2a73...@2.2.3.4:32448?discport=0&raftport=50401"
+      Tm_Url:  2.2.3.4:30371
+      Node_Acct_Addr: 0xB4ba047619946B607dcBE6cA6E54Ff071f95c9E8
+    ```
+
+8. **B** shares its external nodes with **A** by copying and pasting the above output, and sending it
+to **A**.
+
+9. **A** adds the external nodes information from **B** to the **A** configuration file.
+
+    Updated config for **A**:
+
+    ```yaml
+    genesis:
+     consensus: istanbul
+     Quorum_Version: 2.7.0
+     Chain_Id: "1000"
+    nodes:
+    - Node_UserIdent: quorum-node1
+     Key_Dir: key1
+     quorum:
+       quorum:
+         consensus: istanbul
+         Quorum_Version: 2.7.0
+       tm:
+         Name: tessera
+         Tm_Version: 0.10.6
+     geth:
+       Geth_Startup_Params: ""
+    - Node_UserIdent: quorum-node2
+     Key_Dir: key2
+     quorum:
+       quorum:
+         consensus: istanbul
+         Quorum_Version: 2.7.0
+       tm:
+         Name: tessera
+         Tm_Version: 0.10.6
+     geth:
+       Geth_Startup_Params: ""
+    - Node_UserIdent: quorum-node3
+     Key_Dir: key3
+     quorum:
+       quorum:
+         consensus: istanbul
+         Quorum_Version: 2.7.0
+       tm:
+         Name: tessera
+         Tm_Version: 0.10.6
+     geth:
+       Geth_Startup_Params: ""
+    external_nodes:
+     - Node_UserIdent:  quorum-node1
+       Enode_Url: "enode://dca1...@2.2.3.4:31923?discport=0&raftport=50401"
+       Tm_Url:  2.2.3.4:30482
+       Node_Acct_Addr: 0x9712A5aa48A654e9B117467748eF1F793153E918
+     - Node_UserIdent:  quorum-node2
+       Enode_Url: "enode://2a73...@2.2.3.4:32448?discport=0&raftport=50401"
+       Tm_Url:  2.2.3.4:30371
+       Node_Acct_Addr: 0xB4ba047619946B607dcBE6cA6E54Ff071f95c9E8
+    ```
+
+10. **A** generates the update for the Kubernetes network resources. The update leaves the current
+nodes intact, only generating the diff and, updating the configuration files as necessary.
+
+    ```bash
+    > qctl generate network --update
+    ```
+
+11. **A** deploys the updated resources to its cluster.
+
+   ```bash
+   > qctl deploy network
+   ```
+
+   The clusters can now connect and there is a
+   GoQuorum Kubernetes network spanning multiple clusters.
+
+## Multi-cluster across namespaces using NodePorts
+
+A namespace in Kubernetes is an isolated cluster within the same running Kubernetes instance. We will walk through a
+flow of adding nodes across namespaces exposing the services via a NodePort service.
+
+A NodePort service in Kubernetes is a way to expose a pod to the outside world via K8s_Node_IP:Node_Port
+
+The NodePort is not known until deployment time, but can be obtained from the running Kubernetes cluster once it
+is up, the blue port represents the internal port used by the cluster, and the highlighted port represents the port
```

Comment:
Is this referencing an image? Or something the user would see? (ie, the blue port)

Reply:
ah yes in a doc I could add color, this should be changed from:
" the blue port represents the internal port used by the cluster, and the highlighted port represents the port"
to
"NodePorts are represented as pair <Internal_Port:External_Port>, e.g.  in the example below for `quorum-node1` `8545:31346/TCP` the port `8545` represents the internal port used by the cluster, and the `31346` port represents the port.."

Might be helpful to link  "A NodePort service in Kubernetes" to K8s NodePort documentation as well https://kubernetes.io/docs/concepts/services-networking/service/#nodeport.

Reply:
Fixed and link added